### DISCUSSION TITLE
放个试用rrocr key

### DISCUSSION THREAD

**Discussion by WittF** [mikumifa/biliTickerBuy#101] [聊天专区]
不为此key使用问题负责, 要是因为用不了自动过码到时候没抢到别找我嗷
-> **098f6219d15d4680a8caf171421df4e1** <-
rrcor起充10元我一个人用不完, 且试了一下发现不达预期用不上所以放这了

**Comment by YourComputer011** [comment]
能具体讲讲是哪里没达到预期吗，做个参考，谢谢了

**Reply by WittF** [reply]
> 能具体讲讲是哪里没达到预期吗，做个参考，谢谢了

过码挺快, 但是不太稳定, 容易超时连不上(可能是我自己网的问题), 主要是用不完所以就扔这做慈善了, 平时是用capsolver和2captcha的浏览器插件的
![timeout](https://cr.wittf.site/f/EGTp/timeout.png)

**Reply by YourComputer011** [reply]
那请教下是capsolver相对稳定一些吗，平时不太了解这方面

**Reply by WittF** [reply]
> 那请教下是capsolver相对稳定一些吗，平时不太了解这方面

如果要求速度那最好还是手动+rrocr, 我是用capsolver过码会稳点, capsolver会比rrocr慢个5秒

**Reply by WittF** [reply]
capsolver注册就可得1美元试用余额, 你可以试试这两个哪个对你更合适

**Reply by mikumifa** [reply]
capsolver如果调用的话，就没办法手动了，一直卡在那里很难受的，而且没有解决办法，因为一旦把验证码的请求给了capsolver，就没办法再次使用那个验证码请求了

**Comment by Allroha** [comment]
实测可用，感谢！！

**Comment by WittF** [comment]
查询该key余额 ↓
http://api.rrocr.com/api/integral.html?appkey=098f6219d15d4680a8caf171421df4e1

**Reply by WittF** [reply]
还剩4000多点

**Comment by WittF** [comment]
顶一下，刚查了一下还剩7700多，需要的自取

**Reply by liarshen** [reply]
大佬我这显示404错误是为啥

**Reply by liarshen** [reply]
> 大佬我这显示404错误是为啥

不好意思 是我忘记关梯子了  谢谢大佬！
